pkg/ccl/backupccl/testdata/backup-restore/drop-schedule-backup

```diff
@@ -0,0 +1,25 @@
+# These tests validate the DROP SCHEDULE command for dropping the incremental scheduled
+# backup when the full scheduled backup is dropped.
+
+new-server name=s1
+----
+
+exec-sql
+CREATE DATABASE foo;
+CREATE TABLE foo.t (x INT);
+INSERT INTO foo.t VALUES (1), (2), (3);
+----
+
+exec-sql
+CREATE SCHEDULE 'hello' FOR BACKUP DATABASE foo INTO 'userfile:///newbackup' WITH revision_history RECURRING '@daily';
+----
+
```

Comment:
add a SHOW SCHEDULES line to validate that an incremental and a full schedule were created?

Reply:
I expect there to be 2 rows in the schedule's table with the label 'hello'